src/main/java/com/mindee/MindeeClient.java

```diff
@@ -323,6 +324,40 @@ private <T extends Inference> AsyncPredictResponse<T> enqueueAndParse(
     throw new RuntimeException("Max retries exceeded. Failed to get the document.");
   }
 
+  /**
+   * Send a local file to a workflow execution.
+   */
+  public <T extends Inference> WorkflowResponse<T> executeWorkflow(
+      Class<T> type,
+      String workflowId,
+      LocalInputSource localInputSource
+
+  ) throws IOException {
+    return this.mindeeApi.executeWorkflowPost(
+        type,
+        workflowId,
+        RequestParameters.builder()
+            .file(localInputSource.getFile())
+            .fileName(localInputSource.getFilename())
+            .build()
+    );
+  }
+
+  public WorkflowResponse<GeneratedV1> executeWorkflow(
+      String workflowId,
+      LocalInputSource localInputSource
+
+  ) throws IOException {
+    return this.mindeeApi.executeWorkflowPost(
+        GeneratedV1.class,
+        workflowId,
+        RequestParameters.builder()
+            .file(localInputSource.getFile())
+            .fileName(localInputSource.getFilename())
+            .build()
+    );
+  }
```

Comment:
Not sure this is needed at this point. I think it's fine to require the type to always be passed, even for generic.

Reply:
yeah I had thought of only including GeneratedV1 as a first pass (since inference return is currently empty) but forgot to update it